rasa/core/channels/slack.py

```diff
@@ -109,6 +109,20 @@ def _get_text_from_slack_buttons(buttons):
         return super(SlackBot, self).api_call("chat.postMessage", **json_message)
 
 
+def is_app_mention(user_message) -> bool:
```

Comment:
please add tests for these as well as type annotations for the parameters & a doc string

Reply:
I've added the type annotations and standardized these with the other functions in the SlackInput channel. However, I'm not certain how to proceed with the test cases as the current channel tests for slack do not have a token. Is there a mock testing kit you'd prefer that I use for these?